src/ui/server/helpers/summary-lists/your-buyer/company-or-organisation/index.ts

```diff
@@ -26,8 +26,11 @@ const {
  * @returns {Object}
  */
 export const generateAddressObject = (answers: ApplicationBuyer) => {
+  // replace new lines with line breaks to display in summary list
+  const address = answers[ADDRESS].replaceAll('\r\n', '<br />');
```

Comment:
Could this be a helper function so we can easily re-use in other places if required?

Also is there a possible scenario where there is only `\r` and no `\n`, and vice versa?

Reply:
from the component, no.  All line breaks entered into the textarea seem to be` \r\n` if a new line is added